docs/cni/kubernetes/manifests/calico-configmap.yaml

```diff
@@ -0,0 +1,33 @@
+# This ConfigMap can be used to configure a self-hosted Calico installation.
+# See `calico-hosted.yaml` for an example of a Calico deployment which uses
+# the config in this ConfigMap.
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: calico-config 
+  namespace: kube-system
+data:
+  # Configure this with the location of your etcd cluster.
```

Comment:
This comment should also tell people they need to copy this config below in the `cni_network_config`.  (I assume we can't easily refer to this option from there?)